src/ela_did.h

```diff
@@ -2086,13 +2212,32 @@ DID_API bool DIDStore_ContainsPrivateIdentity(DIDStore *store);
  *                             "chinese_traditional", "czech", "english", "french",
  *                             "italian", "japanese", "korean", "spanish".
  * @param
- *      extendedkey            [in] Extendedkey string.
+ *      force                  [in] If private identity exist, remove or remain it.
+ *                              force = true, remove the old identity and initial new identity.
```

Comment:
If force is true, then will choose to create a new identity even if the private identity already exists and the new private key will replace the original one in DID-store.